clients/ethereumjs/wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+client_datadir="~/.ethereum"
+
+client_args=("$@")
+while [[ $# -gt 0 ]]; do
+    case $1 in
+    --dataDir=*)
+        client_datadir="${1#*=}"
+        ;;
+    --dataDir)
+        client_datadir="${2}"
+        ;;
+    esac
+    shift
+done
+
+source /wrapper/wrapper.lib.sh
+
+start_client() {
+    source $testnet_dir/nodevars_env.txt
+
+    ephemery_args=""
+    if [ -z "$(echo "${client_args[@]}" | grep "bootnodes")" ]; then
+        ephemery_args="$ephemery_args --bootnodes=$BOOTNODE_ENODE_LIST"
```

Comment:
we should probably also collect these args in the arg parsing you do above but i guess works here, may be just drop a comment with a future TODO